docker-storage-setup.sh

```diff
@@ -435,11 +438,11 @@ setup_lvm_thin_pool () {
   tpool=`get_configured_thin_pool`
 
   if [ -n "$tpool" ]; then
-     local escaped_pool_lv_name=`echo $POOL_LV_NAME | sed 's/-/--/g'`
+     local escaped_pool_lv_name=`echo $CONTAINER_THINPOOL | sed 's/-/--/g'`
```

Comment:
I figured to allow it to still default to docker-storage if they only specify input.  Simplifies the code and  no huge benefit to the error code.  I guess not allowing them to specify more then 2 is no longer covered, but I don't see that is all that useful.